libs/json-model/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,10 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
maybe use <!--/* To learn more about this file see: https://angular.io/config/tsconfig. */-->